include/si.h

```diff
@@ -110,6 +110,7 @@ typedef enum ScriptOpcode {
 #define SI_ARRAY(v) ((v - 190000000))
 #define SI_ARRAY_FLAG(v) ((v - 210000000))
 #define SI_FIXED(v) (((v * 1024.0f) + -230000000)) // See float_to_fixed_var
+#define ARGS_END 0x80000000
```

Comment:
AFAIK this is only for MakeEntity, and we should probably name it as such. (`make_entity` (the internal non-API function MakeEntity wraps) is a real variable-argument fn using `va_list` and friends, which is a bit scary!)